ui/src/shared/middleware/resizeLayout.js

```diff
@@ -12,5 +13,9 @@ export default function resizeLayout() {
       evt.initEvent('resize', false, true)
       window.dispatchEvent(evt)
     }
+    if (window.location.search.includes('present')) {
```

Comment:
Nice and simple, but I could foresee this becoming conflated with future possible query params that may include the word `present` since it's a simple string `includes`. This slight fragility makes me feel that it would be worthwhile to add a utility function, which I suspect would also be more obvious (and helpful) for iteration, and if imported from a utils file could be easily discoverable. I suggest one that offers more robust parsing. I don't know if it's possible in Middleware via React Router, but if not you could just do it via plain JS: https://stackoverflow.com/questions/901115/how-can-i-get-query-string-values-in-javascript. There's an ES6 solution there, and a Google solution, both of which seem good.

Reply:
oh, true, you're right. laziness on my part.

Reply:
no worries! some of those code snippets should be pretty useful

Reply:
I've written plenty of code similar to that in the past. 😸

Reply:
I'm going to add this library: [query-string](https://www.npmjs.com/package/query-string)

It comes posthumously recommended by Michael Jackson as the replacement for react-router query string manipulation.